syntax = "proto3";

package order;

service OrderService {
  rpc AddOrder (Order) returns (OrderResponse);
  rpc GetOrder (OrderId) returns (Order);
  rpc UpdateOrder (Order) returns (OrderResponse);
}

message Order {
  int32 id = 1;
  int32 user_id = 2;
  string title = 3;
  string description = 4;
}

message OrderId {
  int32 id = 1;
}

message OrderResponse {
  string message = 1;
}
